composer/algorithms/channels_last/README.md

```diff
@@ -88,10 +88,19 @@ If the model weights are instead initialized in NHWC format, PyTorch will automa
 
 We currently implement this method by casting the user’s model to channels-last format (no changes to the dataloader are necessary). When the first convolution operation receives its input activation, it will automatically convert it to NHWC format, after which the memory format will persist for the remainder of the network (or until it reaches a layer that cannot support having channels last).
 
+> ✅ Channels Last Improves Training Speed
+>
+> In our experiments, Channels Last improves the attainable tradeoffs between training speed and the final quality of the trained model.
+> We recommend Channels Last for training convolutional networks.
```

Comment:
This isn't universally true, right? Doesn't it slow down UNet? Should we add a less strong statement here?